### DISCUSSION TITLE
Entire content blocks or pages

### DISCUSSION THREAD

**Discussion by OzzyTheGiant** [saadeghi/daisyui#1797] [Ideas / feature requests]
I've gotten to a point where if I want to build websites for new clients, to speed up my workflow, I want to have a template repo with all the configurations and libraries needed to get up and running right away. To go even further, I would want to have entire pages already made, in different variations if the client wants specific layouts.

The only such free tool I've found for this purpose is Tailblocks.cc, which has a nice list of content blocks but no whole pages and their list is relatively short. The other option I found was FlowBite which has libraries for each framework but to get access to their Marketing/Application/E-Commerce UIs, you need to pay for a license. The price is not that bad, but I'm not looking to pay for that at the moment (maybe later after saving up).

So with that in mind, is this something you guys might think of doing in the future? This would improve my workflow tremendously and could probably help me finish websites in just a few hours, which is great because I refuse to use site builders and Wordpress, yet clients at this point expect ridiculously cheap websites, so rather than fight that demand, I'm going to try to embrace it by reducing my work time, improving my workflow, and have templates ready.

Reason for suggesting it to you guys rather than using those existing tools is because I do like the "less classes" approach compared to those tools.

**Comment by saadeghi** [comment]
Hey,  
daisyUI doesn't provide blocks or templates yet. Currently it just includes smaller pieces of the page and it's up to the developer how to put them together.  

I would like to explore that idea and provide templates in the future but honestly, it's not an easy job to create those many blocks and templates. Taking a look at Tailwind UI or Flowbite, etc you can guess how much time it takes to create them all.  

So despite the fact that I'm interested in this idea, I don't think I would personally have enough time to work on it yet because I have to work on daisyUI itself and my other freelance projects (daisyUI is a nonprofit project after all)  

Even if I get enough time to create those templates and blocks releasing them as free would be really hard for me unless I get big sponsors to support the project financially.

**Reply by OzzyTheGiant** [reply]
Yeah that makes sense, I was just wondering if that maybe would be on the roadmap at some point, even as a paid offering. For me it's more about sticking with Daisy UI rather than migrating to another library, as I have a few websites that depend on it.

**Reply by saadeghi** [reply]
I can say it is possible to happen but I can't promise when... (regarding my current time and financial resources. Because to me, daisyUI package itself is the priority. There are a lot to do and I have to spend time on that) However if somehow I can find other people willing to contribute on that idea it could happen sooner.
